src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -580,10 +577,10 @@ private void onPinnedEndpointChangedEvent(
         // Find the new pinned endpoint.
         String newPinnedEndpointID = (String) jsonObject.get("pinnedEndpoint");
 
-        List<String> newPinnedIDList = Collections.EMPTY_LIST;
+        Set<String> newPinnedIDList = Collections.EMPTY_SET;
```

Comment:
maybe we want to change the variable to something like `newPinnedIDs` (or just `newPinnedEndpoints` like used in `pinnedEndpointsChanged` below)?  also...doesn't look like we get to take advantage of multiple endpoints being pinned here, we only parse a single id?